internal/contracts/contracts.go

```diff
@@ -274,4 +275,31 @@ func (mc *JSONContract) Unmarshal() (Contract, error) {
 		mc.StateNonce,
 	}
 	return c, nil
+	fmt.print(c)
+}
+
+
+
+funcGenerateRandomContract() Contract{
+	b := rand.Read(make([]byte, 32))
+	
+	genSenderPubKey := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+	genrecipPubKeyHash := rand.Read(make([]byte, 32))
+	genVersion := binary.LittleEndian.Uint16(b[:])
+	genSigLen := binary.LittleEndian.Uint8(b[:]);
+	genSignature := rand.Read(make([]byte, genSigLen));
+	genValue := binary.LittleEndian.Uint64(b[:])
+	genStateNonce := binary.LittleEndian.Uint64(b[:])
+	
+	return &Contract{
+		Version:  genVersion,
+		SenderPubKey: genSenderPubKey,
+		SigLen:  genSigLen,
+		Signature:  genSignature,
+		RecipPubKeyHash:  genrecipPubKeyHash,
+		Value:  genValue,
+		StateNonce:  genStateNonce,
+		
+	}, nil
```

Comment:
- [ ] There is no need to return a `nil` here, since your function only returns a `Contract` per the signature.